packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -735,23 +780,62 @@ export class IntervalCollectionView<TInterval extends ISerializableInterval> ext
         return interval;
     }
 
-    public deleteInterval(serializedInterval: ISerializedInterval, local: boolean, op: ISequencedDocumentMessage) {
-        const interval = this.localCollection.removeInterval(serializedInterval.start, serializedInterval.end);
+    public delete(start: number, end: number) {
```

Comment:
>delete [](http://example.com/codeflow?start=11&length=6)

i don't think this is the right behavior. i don't think we should ever loop in delete. i see two cases, delete the interval at the position without id, or with id. i think both these cases should be explicit

Reply:
The behavior we decided on was that delete(start, end) would delete all the intervals that match (start, end), and one delete op would go over the wire (with ID) for each interval. So if client A does delete(start, end), it looks to client B as though client A did delete(id1), delete(id2), ... . That's what this is doing, but I do think it would be nicer to have findInterval return an array and then have delete(start, end) loop over the array.